unfetter-discover/unfetter
# ntctf data for processor

## To Test
* pull this PR
* restart stack
* open a mongo shell
* verify > 0 attackpatterns `db.getCollection('stix').find({'stix.type': 'attack-pattern', 'stix.kill_chain_phases.kill_chain_name': { $not: { $eq: 'mitre-attack' } }}).count()`
* browse a few `db.getCollection('stix').find({'stix.type': 'attack-pattern', 'stix.kill_chain_phases.kill_chain_name': { $not: { $eq: 'mitre-attack' } }})`
